website/content/docs/connect/observability/ui-visualization.mdx

```diff
@@ -27,7 +27,7 @@ URLs](#configuring-dashboard-urls).
 It is possible to configure the UI to fetch basic metrics from your metrics
 provider storage to augment the visualization as displayed below.
 
-![Consul UI Service Mesh Visualization](/img/ui-service-topology-view.png)
+![Consul UI Service Mesh Visualization](/img/ui-service-topology-view-hover.png)
```

Comment:
This image is not rendering. I don't see this image added to the `website/public/img/` directory.

Reply:
Yup my bad, I forgot to commit that folder! Updating now